src/main/java/by/gexateq/simplerestservice/entity/Employee.java

```diff
@@ -0,0 +1,23 @@
+package by.gexateq.simplerestservice.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "employees")
+public class Employee {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String first_name;
```

Comment:
properties names should be in camel case as usual. By default it will be mapped to corresponded fields e.g. firstName property -> first_name column.